opm/autodiff/StandardWell_impl.hpp

```diff
@@ -2777,4 +2799,15 @@ namespace Opm
             OpmLog::debug(msg);
         }
     }
+
+
+
+
+
+    template<typename TypeTag>
+    void
+    StandardWell<TypeTag>::
+    updateWaterThroughput(const double dt, WellState &well_state) const
```

Comment:
Should tag the arguments as unused, and the `&` should be adjacent to the type.

Also: why cannot the throughput be computed for the standard and multisegment well models?

Reply:
More like we do not have the information of `perf_water_vel` for producers. 

When polymer molecular weight is involved, injector will be `StandardWellV`, producer will be `StandardWell`. We do not know how to handle multi-segment well yet.